docs/tornjak-quickstart.md

```diff
@@ -0,0 +1,318 @@
+
+➜  quickstart git:(master) kubectl exec -n spire spire-server-0 -- \
+    /opt/spire/bin/spire-server entry create \
+    -spiffeID spiffe://example.org/ns/spire/sa/spire-agent \
+    -selector k8s_sat:cluster:demo-cluster \
+    -selector k8s_sat:agent_ns:spire \
+    -selector k8s_sat:agent_sa:spire-agent \
+    -node
+Entry ID         : 03d0ec2b-54b7-4340-a0b9-d3b2cf1b041a
+SPIFFE ID        : spiffe://example.org/ns/spire/sa/spire-agent
+Parent ID        : spiffe://example.org/spire/server
+Revision         : 0
+TTL              : default
+Selector         : k8s_sat:agent_ns:spire
+Selector         : k8s_sat:agent_sa:spire-agent
+Selector         : k8s_sat:cluster:demo-cluster
+
+➜  quickstart git:(master) kubectl exec -n spire spire-server-0 -- \
+    /opt/spire/bin/spire-server entry create \
+    -spiffeID spiffe://example.org/ns/default/sa/default \
+    -parentID spiffe://example.org/ns/spire/sa/spire-agent \
+    -selector k8s:ns:default \
+    -selector k8s:sa:default
+Entry ID         : 11a367ab-7095-4390-ab89-34dea5fddd61
+SPIFFE ID        : spiffe://example.org/ns/default/sa/default
+Parent ID        : spiffe://example.org/ns/spire/sa/spire-agent
+Revision         : 0
+TTL              : default
+Selector         : k8s:ns:default
+Selector         : k8s:sa:default
+
+➜  quickstart git:(master) kubectl apply -f client-deployment.yaml
+deployment.apps/client created
+
+➜  quickstart git:(master) kubectl exec -it $(kubectl get pods -o=jsonpath='{.items[0].metadata.name}' \
+   -l app=client)  -- /bin/sh
+/opt/spire # /opt/spire/bin/spire-agent api fetch -socketPath /run/spire/sockets/agent.sock
+Received 1 svid after 8.8537ms
+
+SPIFFE ID:		spiffe://example.org/ns/default/sa/default
+SVID Valid After:	2021-04-06 20:13:02 +0000 UTC
+SVID Valid Until:	2021-04-06 21:13:12 +0000 UTC
+CA #1 Valid After:	2021-04-06 20:12:20 +0000 UTC
+CA #1 Valid Until:	2021-04-07 20:12:30 +0000 UTC
+
+/opt/spire #
+```
+
+
+## Step 2: Configuring Tornjak
+
+Now that we have the SPIRE deployment set up, it should be fairly simple to use Tornjak.
+
+### Creating the Tornjak Configmap
+
+We first need to create the configmap. We can create a new file: 
+
+```
+➜  quickstart git:(master) k cat tornjak-configmap.yaml 
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: tornjak-agent
+  namespace: spire
+data:
+  server.conf: |
+    server {
+      metadata = "insert metadata"
+    }
+
+    plugins {
+      DataStore "sql" {
+        plugin_data {
+          drivername = "sqlite3"
+          filename = "./agentlocaldb"
+        }
+      }
+
+    }
+```
+
+### Updating the SPIRE Server statefulset
+Next, we need to update the image of the SPIRE server statefulset, as well as make sure we pass in the Tornjak config. The statefulset will look something like this, where we have commented on the changed or new lines: 
+
+```
+➜  quickstart git:(master) k cat server-statefulset.yaml 
```

Comment:
Since we never talked about the alias "k", I suggest we replace all single `k` to `kubectl`